ui/src/pages/backoffice/components/Sidebar/AdminMenu.js

```diff
@@ -0,0 +1,22 @@
+import { FrontSiteRoutes } from '@routes/urls';
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Divider, Menu } from 'semantic-ui-react';
+
+export default class AdminMenu extends Component {
+  render() {
+    return (
+      <>
+        <Divider horizontal>Admin menu</Divider>
+        <Menu text vertical className="bo-menu">
+          <Menu.Item as={'a'} href="https://localhost:5000/admin">
```

Comment:
We cannot hardcode `localhost` here, I would add a new route in `urls.js`

Reply:
I would suggest to create a `baseUrl` similar to [this](https://github.com/inveniosoftware/invenio-app-ils/blob/master/ui/src/api/base.js#L5)

Reply:
> I would suggest to create a `baseUrl` similar to [this](https://github.com/inveniosoftware/invenio-app-ils/blob/master/ui/src/api/base.js#L5)

Right, better, because of the 5000 port, otherwise it won't be needed...